docker-compose-qgis-server.yml

```diff
@@ -0,0 +1,19 @@
+version: '2'
+services:
+
+  qgis-server:
+    image: kartoza/geonode_qgis-server
```

Comment:
I put a sample docker-compose file to use QGIS Server easily as docker image in Travis.
I preferred this way over installing QGIS Server natively on the environment, because it takes additional steps to configure.
However, am I allowed to use Kartoza's docker repo, or do I have to create it on GeoNode's docker repo (I'm not sure if I'm allowed/able to do that).